File: problem_solving_for_beginners/string_problems/c-programming/str_concat.c
#include<stdio.h>
/**
 * main - Concatenates two strings
 * 
 * Return: 0 always 
 */
int main(void)
{
    char str1[10], str2[10], str3[20];
    
    printf("Enter two strings: ");
    scanf("%s%s", str1, str2);
    
    int i, j;
    
    for (i = 0; str1[i] != '\0'; i++)
        str3[i] = str1[i];
        
    for (j = 0; str2[j] != '\0'; j++)
        str3[i+j] = str2[j];
        
    str3[i+j] = '\0';
    
    printf("%s", str3);
    
    return 0;
}
    

File: problem_solving_for_beginners/string_problems/python/str_concat.py
#!/usr/bin/python3
str1 = input("Enter first string: ")
str2 = input("Enter second string: ")
str3 = ''

"""String concatenation"""
for ch in str1:
    str3 += ch

for ch2 in str2:
    str3 += ch2
    
print(str3)
